src/org/thoughtcrime/securesms/ProfileDocumentsFragment.java

```diff
@@ -177,6 +177,7 @@ protected void setCorrectMenuVisibility(Menu menu) {
     menu.findItem(R.id.details).setVisible(singleSelection);
     menu.findItem(R.id.show_in_chat).setVisible(singleSelection);
     menu.findItem(R.id.save).setVisible(singleSelection);
+    menu.findItem(R.id.share).setVisible(false);
```

Comment:
documents can never be shared - is that intended?

Reply:
@r10s ups! for some reason I had the wrong memory that it was not possible to share a document when selecting it in the conversation, I just checked and indeed it is possible! fixing now

Reply:
thanks for re-checking!